package.json

```diff
@@ -8,19 +8,21 @@
     "dist"
   ],
   "scripts": {
-    "build": "rimraf dist && babel src -d dist",
-    "docs": "gulp docs",
+    "docs": "gulp",
+    "build": "npm run build:dist && npm run build:docs",
+    "build:dist": "rimraf dist && babel src -d dist",
+    "build:docs": "gulp docs",
     "clean": "while read line; do rm -rf $line; done < .gitignore",
-    "predeploy:docs": "npm run docs",
+    "predeploy:docs": "npm run build:docs",
     "deploy:docs": "gh-pages -d docs/build -m 'deploy docs [ci skip]'",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
     "lint:watch": "watch 'npm run lint --silent' docs gulp src test",
-    "prepublish": "npm run build",
+    "prepublish": "npm run build:dist",
     "release:major": "ta-script npm/release.sh major",
     "release:minor": "ta-script npm/release.sh minor",
     "release:patch": "ta-script npm/release.sh patch",
-    "start": "gulp",
+    "start": "npm run docs",
```

Comment:
I think `npm start` should always start the project without a need to understand the underlying implementation.  Also, if we drop `gulp` (which we hope to do), `npm start` should still start the project.  If we tie the start script to gulp, then there is unnecessary coupling that would require updates to the README and dev workflow since they'll need to run a new command.

`npm start` might not always run the doc site, it will always "start the project".  There is an `npm run docs` that will always run the doc site.

This is my rationale at least.  Open to change if there is pretty good consensus that thinks otherwise.